File: DSA 1/7.c
#include<stdio.h>

int main(){
    int temp;
    int arr[5] = {0, 4, 0, 6, 1};
    int count = 0; // Counter for non-zero elements

    // Move non-zero elements to the front
    for(int i = 0; i < 5; i++){
        if(arr[i] != 0){
            arr[count] = arr[i];
            count++;
        }
    }

    // Fill the remaining elements with zeros
    for(int i = count; i < 5; i++){
        arr[i] = 0;
    }

    printf("Modified Array: ");
    for(int i = 0; i < 5; i++){
        printf("%d ", arr[i]);
    }

    return 0;
}

File: DSA_2/7.js
function check(arr){
    if(arr[0]<arr[1]){
        for(let i=0;i<(arr.length-1);i++){
            if(arr[i]>arr[i+1]){
                return false
            }
        }
        return true
    }
    else{
        for(let i=0;i<(arr.length-1);i++){
            if(arr[i]<arr[i+1]){
                return false
            }
        }
        return true
    }
}


let arr=[1,2,3,3,4]
console.log(check(arr))